src/pages/[...slug].tsx

```diff
@@ -23,12 +24,11 @@ const Templates: NextPage<PageProps> = ({
   templates,
   code,
 }) => {
+  const seoTitle = `${category.name}/${subCategory.name} - Chakra Templates`;
```

Comment:
You could use `SEO_TITLE` from the constants instead of the `Chakra Templates` here

Reply:
True